### PR TITLE
instruction view screen

### PR DESCRIPTION
**Pull request by yash6102000**

### CODE REVIEW

File: app/components/assessment/CandidateInstruction.tsx [hunk 1]
```diff
@@ -2,13 +2,16 @@ import { useLoaderData, useSubmit } from '@remix-run/react'
 import {
   candidateExamConstants,
   routeFiles,
+  sectionsConstants,
 } from '~/constants/common.constants'
 import type { SectionInTest, TestSection } from '~/interface/Interface'
 import Button from '../form/Button'
-import sanitizeHtml from 'sanitize-html'
+// import sanitizeHtml from 'sanitize-html'
```
Comment (arpita-20) [inline app/components/assessment/CandidateInstruction.tsx:9]: remove commented code

File: app/components/assessment/CandidateInstruction.tsx [hunk 2]
```diff
@@ -23,64 +26,132 @@ const CandidateInstruction = () => {
...
   return (
-    <div className="h-full flex-col overflow-y-auto">
-      <div>
-        <div
-          className="pb-9 text-base text-gray-700"
-          dangerouslySetInnerHTML={{
-            __html: sanitizeHtml(instructions?.test?.description),
-          }}
-        />
+    // <div className="h-full flex-col overflow-y-auto">
```
Comment (arpita-20) [inline app/components/assessment/CandidateInstruction.tsx:31]: remove commented code

File: app/components/assessment/CandidateInstruction.tsx [hunk 3]
```diff
@@ -23,64 +26,132 @@ const CandidateInstruction = () => {
...
+    //   </div>
+    // </div>
+
+    <div className="flex flex-col items-center justify-center gap-16 pt-12">
+      <div className="flex flex-col items-center justify-center gap-4">
+        <h3 className="text-2xl font-bold text-gray-900">
+          {candidateExamConstants.candidateInsWelcome} {candidate.firstName}
+        </h3>
+        <p className="text-base font-medium text-gray-500">
+          Lorem Ipsum is simply dummy text of the printing and typesetting
```
Comment (arpita-20) [inline app/components/assessment/CandidateInstruction.tsx:96]: add text in common constants

File: app/routes/assessment/$assessmentId/instructions.tsx [hunk 4]
```diff
@@ -1,9 +1,10 @@
 import type { ActionFunction, LoaderFunction } from '@remix-run/node'
 import { json } from '@remix-run/node'
 import { redirect } from '@remix-run/node'
-import { useLoaderData } from '@remix-run/react'
+// import { useLoaderData } from '@remix-run/react'
```
Comment (arpita-20) [inline app/routes/assessment/$assessmentId/instructions.tsx:4]: remove commented code

**Review by anuragP04 — APPROVED** [review]

**Pull request merged**